pytorch_lightning/accelerators/ipu.py

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from collections import Callable
+from collections.abc import Callable
 from typing import Any
```

Comment:
This was probably the intention

```suggestion
from typing import Any, Callable
```

Reply:
agreed, we should import from typing